pkg/api/unversioned/types.go

```diff
@@ -339,6 +339,8 @@ type APIResource struct {
 	Name string `json:"name"`
 	// namespaced indicates if a resource is namespaced or not.
 	Namespaced bool `json:"namespaced"`
+	// kind is the kind for the resource (e.g. 'Foo' is the kind for a resource 'foo')
```

Comment:
> This leaks internal go type names into our API. @bgrant0607

`kind` isn't necessarily the same as the go type name.  This is `kind` in the `RESTMapper` sense that uses the `kind` in the JSON serialization.  Or at least that was the intent.